gtk/MakeDialog.cc

```diff
@@ -347,12 +347,18 @@ void MakeDialog::Impl::updatePiecesLabel()
     }
     else
     {
-        gstr += gtr_sprintf(
-            ngettext("%1$s; %2$'d File", "%1$s; %2$'d Files", builder_->fileCount),
-            tr_strlsize(builder_->totalSize),
-            builder_->fileCount);
+        gstr += fmt::format(
+            ngettext("{complete_size}; {file_count:L} File", "{complete_size}; {file_count:L} Files", builder_->fileCount),
```

Comment:
Have been wondering for some time now: is capitalisation of "File" and "Piece" in the middle of sentence necessary here? It's not a title, which may have explained it in English, and I notice that some other languages drop it (athough they mey have different rules).

Reply:
According to the [HIG](https://teams.pages.gitlab.gnome.org/Design/hig-www/guidelines/writing-style.html#capitalization) this would count as sentence capitalization so it shouldn't be capitalized here. I've changed the case in 33e5452075280c155db52a4e32103f9db88b1e71